core/src/main/scala/org/apache/spark/rdd/SqlNewHadoopRDD.scala

```diff
@@ -96,6 +96,11 @@ private[spark] class SqlNewHadoopRDD[V: ClassTag](
 
   @transient protected val jobId = new JobID(jobTrackerId, id)
 
+  // If true, enable using the custom RecordReader for parquet. This only works for
+  // a subset of the types (no complex types).
+  protected val enableUnsafeRowParquetReader: Boolean =
+      sc.conf.getBoolean("spark.parquet.enableUnsafeRowRecordReader", true)
```

Comment:
Yea, reynold suggested the same thing. It requires moving this entire class to that package. I was going to do it in the next PR.